db.js

```diff
@@ -1,37 +1,35 @@
-const sqlite3 = require('sqlite3').verbose();
-const db = new sqlite3.Database('./data/db.db');
-
-const init = () => {
-        db.serialize(() => {
-            db.run(`CREATE TABLE IF NOT EXISTS players (
+const sqlite = require('sqlite-async');
+let db;
+const init = async () => {
+    try {
+        db = await sqlite.open('./data/db.db');
+        await db.run(`CREATE TABLE IF NOT EXISTS players (
             id INT,
             name STRING,
-            hand_elo
+            hand_elo INT,
+            UNIQUE(id)
             )`);
-            const stmt = db.prepare("INSERT INTO lorem VALUES (?)");
-            //     stmt.run("Ipsum " + i);
-            stmt.finalize();
-            db.each("SELECT rowid AS id, info FROM lorem", function (err, row) {
-                console.log(row.id + ": " + row.info);
-            });
-        });
+        console.log('DB started');
+    } catch (error) {
+        throw Error('can not access sqlite database');
+    }
 };
 
-const insertPlayerIfNotExists = (memberId, name, hand_elo = 5) => {
-    const stmt = db.prepare(`INSERT OR IGNORE INTO players(id, name, hand_elo)
+const insertPlayerIfNotExists = async (memberId, name, hand_elo = 5) => {
+    const stmt = await db.prepare(`INSERT OR IGNORE INTO players(id, name, hand_elo)
```

Comment:
the previous code block is creating a table with the following columns: id, name, elo, unique id and here you are trying to insert olny 3 values